app/main.py

```diff
@@ -2,0 +1,30 @@
+import os
+import shutil
+
+
+def move_file(command: str) -> None:
+    command_parts = command.split()
+
+    if len(command_parts) != 3 or command_parts[0] != "mv":
+        raise ValueError(
+            "Command must be in the format 'mv source destination'"
+        )
+
+    source = command_parts[1]
+    destination = command_parts[2]
+
+    if not source or not destination:
+        raise ValueError("'source' and 'destination' cannot be empty")
+
+    if not os.path.exists(source):
+        raise FileNotFoundError(f"File {source} does not exist")
+
+    if destination.endswith("/"):
+        os.makedirs(destination, exist_ok=True)
+        destination = os.path.join(destination, os.path.basename(source))
```

Comment:
Code Style: It's recommended to use `os.path.join` to concatenate parts of the path. This will make your app cross-platform and will use either `/` or `\` depending on the OS. Here, you could use `os.path.join(destination, os.path.basename(source))` even if `destination` doesn't end with `/`.